snc.sh

```diff
@@ -226,14 +226,15 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo mv /home/core/kubeconfig
 
 # Export all manifests to the disk, modify them and use them in the CVO
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo mkdir /opt/release-manifests/'
+retry ${OC} -n openshift-cluster-version get pods -o=name
```

Comment:
log should be `Add missing retry for oc commands` (misplaced 'missing'). This is unrelated to this PR right? The CI issue would happen even without the previous changes?

Reply:
yes this is unrelated to the PR but CI actually failed twice on this one.